src/azure-firewall/azext_firewall/vendored_sdks/v2019_04_01/models/_models.py

```diff
@@ -3182,7 +3182,7 @@ class BGPCommunity(Model):
     :type community_name: str
     :param community_value: The value of the bgp community. For more
      information:
```

Comment:
I suggest we back out changes under folders of `verndored_sdks` as those are generated files from swagger specs, and the next round of code-gen will wipe out the change from this PR. If we like to address the root cause, we should update [specs](https://github.com/Azure/azure-rest-api-specs/tree/master/specification).

Reply:
@yugangw-msft If everything is generated from the Swagger specs, then this PR should be closed and we can address the root problem (which is the Swagger.) Otherwise feel free to merge it - I've taken an item to look into hardcoded `en-us` in Swagger.